R/model-validation.R

```diff
@@ -52,15 +55,15 @@
 #' log_scores <- enw_score_nowcast(summarised_nowcast, obs, log = TRUE)
 #' summarise_scores(log_scores, by = "location")
 enw_score_nowcast <- function(nowcast, latest_obs, log = FALSE,
-                              check = FALSE, round_to = 3, ...) {
+                              check = FALSE, round_to = 3, ..., copy = TRUE) {
   if (!requireNamespace("scoringutils")) {
     stop("scoringutils is required for this function to work")
   }
   long_nowcast <- enw_quantiles_to_long(nowcast)
   if (!is.null(long_nowcast[["mad"]])) {
     long_nowcast[, "mad" := NULL]
   }
-  latest_obs <- coerce_dt(latest_obs)
+  latest_obs <- coerce_dt(latest_obs, copy = copy)
```

Comment:
also never a situation where it doesnt need to copy I think (we can go for less copy args and add if and when people want them?